src/intentLogic/latestPodcast.js

```diff
@@ -53,4 +49,6 @@ const generatePodcastSpeech = (results, isNewIntent) => {
   return ack + buildLatestPodcastSpeech() + podcastTitles + followupQuestion()
 }
 
-const removePodcastFromTitle = (title) => title.replace(/(-|–)?\s?podcast$/, '')
+const getPodcastSeriesName = (tags) => {
+  return tags.filter(tag => tag.podcast).map(t => t.webTitle.split(/[-–]/)[0])
```

Comment:
I know it was me that suggested `filter`, but I've just realised what we're doing here. Would it be better to use [find](http://www.w3schools.com/jsref/jsref_find.asp), since we only want one tag?